flocker/testtools/__init__.py

```diff
@@ -47,11 +47,15 @@
 from twisted.trial.unittest import TestCase
 from twisted.python.logfile import LogFile
 
+from ._flaky import flaky
 from .. import __version__
 from ..common.script import (
     FlockerScriptRunner, ICommandLineScript)
 
 
+# Export for other places to import. Be still, flake8.
+flaky
+
 # This is currently set to the minimum size for a SATA based Rackspace Cloud
```

Comment:
gotcha, which explains the comment just above.
my Syntastic vim checker seems to eat it too.
if vim is happy I am happy